core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala

```diff
@@ -97,4 +97,34 @@ class TopicCommandTest extends ZooKeeperTestHarness with Logging {
     }
     assertFalse("Delete path for topic shouldn't exist after deletion.", zkClient.exists(deleteOffsetTopicPath))
   }
+
+  @Test
+  def testCreateAlterTopicWithRackAware() {
+    val brokers = 0 to 5
+    TestUtils.createBrokersInZk(zkClient, brokers)
+
+    val createOpts = new TopicCommandOptions(Array(
+      "--partitions", "18",
+      "--replication-factor", "3",
+      "--rack-locator-class", "kafka.admin.SimpleRackLocator",
+      "--rack-locator-properties", "0=rack1,1=rack2,2=rack2,3=rack1,4=rack3,5=rack3",
```

Comment:
rack-locator might be a bit confusing to the user when just coming and looking at the new command  / api changes in a release. why not rack-aware or rack-placement-class (keep the SimplaRackLocator class as is) and then rack-placement-properties? or something of the sort?